nengo/tests/test_config.py

```diff
@@ -298,7 +299,11 @@ def test_classparams_str_repr():
 def test_config_repr():
     """tests the repr function in Config class"""
     model = nengo.Network()
-    assert repr(model.config) == "<Config(Connection, Ensemble, Node, Probe)>"
+    print(repr(model.config))
```

Comment:
Leftover print statement, or is this intentional?

Reply:
Just leftover